src/lsdb/core/crossmatch/abstract_crossmatch_algorithm.py

```diff
@@ -9,7 +11,8 @@
 class AbstractCrossmatchAlgorithm(ABC):
     """Abstract class used to write a crossmatch algorithm"""
 
-    DISTANCE_COLUMN_NAME = "_DIST"
+    extra_columns: Dict[str, pd.Series] | None = None
```

Comment:
1. I believe that this `Union` syntax was introduced by Python 3.10
2. I think it would be better to use one of `meta` types supported by `dask` (see [this blog post](https://blog.dask.org/2022/08/09/understanding-meta-keyword-argument)), and do not introduce a new one. Could it be just `Dict[str, type]` or an empty `pd.DataFrame`?

Reply:
Thanks!